source/logs/retention/size.txt

```diff
@@ -0,0 +1,95 @@
+.. _mongosh-log-retention-gb:
+
+===============================
+Modify Maximum Log Storage Size
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can specify a maximum combined storage size for |mdb-shell| log
+files. If the total size of all log files exceeds the maximum, log files
+are deleted until the combined size is below the threshold, starting
+with the oldest log files. By default, there is no maximum log storage
+size.
+
+About this Task
+---------------
+
+To specify a maximum log storage size, set the ``logRetentionGB``
+configuration option. ``logRetentionGB`` can be any positive float
+value (including less than ``1``).
+
+.. include:: /includes/log-retention-multiple-criteria.rst
+
+To check the current storage size of log files, check the size of the
+log folder. To see the current log folder, run the following command
+from |mdb-shell|:
+
+.. code-block:: javascript
+
+   config.get('logLocation')
+
+Before you Begin
+----------------
+
+To check the current maximum log storage size, run the following
+command:
+
+.. code-block:: javascript
+
+   config.get("logRetentionGB")
+
+Steps
+-----
+
+To modify maximum log storage size, set the ``logRetentionGB``
+configuration option. You can set configuration options in the
+:ref:`configuration API <configure-settings-api>` or a
+:ref:`configuration file <configure-settings-global>`.
+
+Modify Maximum Log Storage Size with the API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command uses the config API to set the maximum log storage
+size to 3.5 GB:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: javascript
+
+      config.set("logRetentionGB", 3.5)
+
+   .. output::
+      :language: javascript
+
+      Setting "logRetentionGB" has been changed
+
+Modify Maximum Log Storage Size with a Configuration File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following configuration file sets the maximum log storage size to
+3.5 GB:
+
+.. code-block:: yaml
+
+   mongosh:
+     logRetentionGB: 3.5
+
+Disable Maximum Log Storage Size
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To instruct |mdb-shell| to not delete logs based on storage size, set
+``logRetentionGB`` to ``Infinity``. You can perform this action through
```

Comment:
I do think that this is nicer for consistency with the rest of them so I think it's fine to leave it as is but just fyi `undefined` (the unset value) would also do this. 

it'd be more performant with `undefined` at the moment but this is something we can quickly fix to work the same with `Infinity` so I don't mind leaving it here.
cc: @addaleax @alenakhineika

Reply:
Got it - Thanks for the context and for filing the ticket.